### PR TITLE
send nodeinfo: outboundIP, mac and uname -a

### PR DESCRIPTION
**Pull request by nicboul**

### CODE REVIEW

File: cmd/netvfy-agent/main.go [hunk 1]
```diff
@@ -140,6 +140,18 @@ func genMAC() []byte {
 	return mac
 }
 
+func getOutboundIP() string {
+	conn, err := net.Dial("udp", "8.8.8.8:80")
```
Comment (si74) [inline cmd/netvfy-agent/main.go:144]: can we parametrize this instead of hardcoding IP?

File: cmd/netvfy-agent/main.go [hunk 2]
```diff
@@ -140,6 +140,18 @@ func genMAC() []byte {
...
+	if err != nil {
+		elog.Fatalf("failed get the outbound IP: %v\n", err)
+	}
+	defer conn.Close()
+
+	localAddr := conn.LocalAddr().(*net.UDPAddr)
```
Comment (si74) [inline cmd/netvfy-agent/main.go:150]: ohh hmmm is this the only way to get outgoing IP? i.e. by dialing google DNS server?

Comment (nicboul) [reply cmd/netvfy-agent/main.go:150]: well, any internet IP can do, it's just to force a looking to the routing table

Comment (si74) [reply cmd/netvfy-agent/main.go:150]: understood - just pointing that out if we're actually allocating the IP to the tap interface within this code, I'd presume it's actually stored somewhere -i.e. some variable, etc - and doesn't necessarily need to be looked up?

**Review by si74 — APPROVED** [review]
👍 🚢

**Pull request merged**